addon/components/detail-learner-list.hbs

```diff
@@ -0,0 +1,12 @@
+<ul class="detail-learner-list" data-test-detail-learner-list>
+  {{#each (sort-by "fullName" (await @learners)) as |user|}}
+    {{#if @isManaging}}
+      <li role="button" {{on "click" (fn @remove user)}}>
```

Comment:
Can this just be a button inside of the LI? Instead of the role?

Reply:
i agree on the principle, but in practice this would (and should) be a larger undertaking. we should look and address this in all of those tag-list-style components.

![Selection_097](https://user-images.githubusercontent.com/1410427/77565562-a1a4f380-6e81-11ea-9e2e-5a5c4bb623d0.png)